docs/SetupDatabaseForAKS.md

```diff
@@ -0,0 +1,57 @@
+# Set up a database for Scalar DB/Scalar DL deployment in AKS
+
+This guide explains how to set up a database for Scalar DB/Scalar DL deployment in AKS.
+
+## Cosmos DB
+
+In this section, you will create a Cosmos DB account.
+
+### Requirements
+
+* You must create a Cosmos DB account with the `Core(SQL)` API.
+* You must change Cosmos DB `Default consistency` to `Strong`.
+* You must use Capacity mode as `Provisioned throughput`.
+
+### Recommendations
+
+* You should configure the backup policy as `Continuous` for Point-in-time-restore (PITR).
+* You should configure the service endpoint based on the [Azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/how-to-configure-vnet-service-endpoint) to restrict access to Cosmos DB for production.
+
+### Steps
+
+* Create an Azure Cosmos DB account based on the [Azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/create-cosmosdb-resources-portal#create-an-azure-cosmos-db-account) with the above requirements.
+
+#### Scale Performance
```

Comment:
I think point 2 is the only one step for scale performance, other 2 points are just explanations.

If it is under the steps we need to follow the same format.
And we can specify the explanations as a note.

Reply:
@scalar-boney 
Thank you, I will update that section as you mentioned.

Reply:
The current doc structure is very weird. Users don't get the meaning of indentation.
Why don't you make scaling thing and monitoring optional steps? (make them ones of steps and mark them as `(optional)` or something.)

Reply:
I have updated  the document and made scaling and monitoring as optional steps.